### DISCUSSION TITLE
IP PTZ Camera CPU: AK3918EN080 V200 flash: XM25QH64A

### DISCUSSION THREAD

**Discussion by bolt1502** [roleoroleo/yi-hack-Allwinner-v2#313] [Q&A]
Hello.Can you help with such a camera? CPU: AK3918EN080 V200 flash: XM25QH64A
![DEPBA-ZXJ9c](https://user-images.githubusercontent.com/6560084/143981341-f1fec2bf-2dc5-4161-bd30-f98c11f9431f.jpg)
![S20211127_0002](https://user-images.githubusercontent.com/6560084/143981344-ce5649ac-2825-46b7-9c28-466eab32a372.jpg)
![S20211127_0005](https://user-images.githubusercontent.com/6560084/143981346-54098f30-5043-4e98-9abb-630fbca33bb3.jpg)

There is part of the firmware
[1.zip](https://github.com/roleoroleo/yi-hack-Allwinner-v2/files/7622719/1.zip)
Log UART
[лог UART.txt](https://github.com/roleoroleo/yi-hack-Allwinner-v2/files/7622722/UART.txt)
Firmware version: 6.0.02.10_202108301657
The camera was unsuccessful, there are no copies

**Comment by bolt1502** [comment]
dump is corrupted 
[XM25QH64A@SOIC8.zip](https://github.com/roleoroleo/yi-hack-Allwinner-v2/files/7622736/XM25QH64A%40SOIC8.zip)

**Reply by MAM2022** [reply]
Greetings... did you manage to get a copy of fimware?
if yes, please give me a copy, Because I have the same problem.
Thanks

**Comment by roleoroleo** [comment]
Sorry, but this cpu is not supported by my hacks.
It requires a completely new hack.

**Comment by roleoroleo** [comment]
Pay attention: your dump is corrupted.

**Reply by bolt1502** [reply]
I know, so I addressed you.

**Reply by bolt1502** [reply]
Incorrectly set up the programmer, now the chamber is dead

**Comment by ghost** [comment]
This is the closest I could find https://kms.app/archives/290/ might be able to get some help there. FWIW I have an icsee camera with the Anyka v300, seems to work quite well oob.
